docs/core/integration-quality-scale/rules/iqs006.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "IQS006: Use ConfigEntry.runtime_data to store runtime data"
+---
+
+IQS006 is as follows:
+
+Use ConfigEntry.runtime_data to store runtime data
+
+## Reasoning
+
+The `ConfigEntry` object has a `runtime_data` attribute that can be used to store runtime data.
+This is useful for storing data that is not persisted to the configuration file, but is needed during the lifetime of the configuration entry.
+
+By using `runtime_data`, we maintain a consistency for developers to store runtime data in a consistent and typed way.
+Because of the added typing, we can use tooling to avoid typing mistakes.
```

Comment:
```suggestion
By using `runtime_data`, we maintain consistency for developers to store runtime data in a consistent and typed way.
Because of the added typing, we can use tooling to avoid typing mistakes.
```